[2024-02-26T13:57:28.792+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date permission to 509
[2024-02-26T13:57:28.795+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date'
[2024-02-26T13:57:28.827+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date permission to 509
[2024-02-26T13:57:28.828+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date'
[2024-02-26T13:57:29.680+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_dag1.print_currnt_date scheduled__2024-01-03T00:30:00+00:00 [queued]>
[2024-02-26T13:57:29.688+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_dag1.print_currnt_date scheduled__2024-01-03T00:30:00+00:00 [queued]>
[2024-02-26T13:57:29.688+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-02-26T13:57:29.703+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): print_currnt_date> on 2024-01-03 00:30:00+00:00
[2024-02-26T13:57:29.710+0000] {standard_task_runner.py:57} INFO - Started process 397 to run task
[2024-02-26T13:57:29.712+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'python_dag1', 'print_currnt_date', 'scheduled__2024-01-03T00:30:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/02-python-operator.py', '--cfg-path', '/tmp/tmp2ws85yzr']
[2024-02-26T13:57:29.714+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask print_currnt_date
[2024-02-26T13:57:29.752+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date permission to 509
[2024-02-26T13:57:29.752+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag1/run_id=scheduled__2024-01-03T00:30:00+00:00/task_id=print_currnt_date'
[2024-02-26T13:57:29.754+0000] {task_command.py:410} INFO - Running <TaskInstance: python_dag1.print_currnt_date scheduled__2024-01-03T00:30:00+00:00 [running]> on host 2fe49a870f57
[2024-02-26T13:57:29.834+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='heumsi' AIRFLOW_CTX_DAG_ID='python_dag1' AIRFLOW_CTX_TASK_ID='print_currnt_date' AIRFLOW_CTX_EXECUTION_DATE='2024-01-03T00:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-03T00:30:00+00:00'
[2024-02-26T13:57:29.835+0000] {logging_mixin.py:150} INFO - 2024-02-26는 월요일입니다.
[2024-02-26T13:57:29.836+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-02-26T13:57:29.845+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=python_dag1, task_id=print_currnt_date, execution_date=20240103T003000, start_date=20240226T135729, end_date=20240226T135729
[2024-02-26T13:57:29.884+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-02-26T13:57:29.899+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
